packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -442,6 +458,9 @@ export class RunningSummarizer implements IDisposable {
     }
 
     private async summarizeCore(reason: string, safe: boolean): Promise<boolean | undefined> {
+        const summarizeCount = ++this.summarizeCount;
+        this.summaryLogger = new ChildLogger(this.summarizerLogger, "Summary", { summarizeCount });
```

Comment:
Can we call them "baseLogger" and "currentLogger"?
summary vs. summarizer does not tell me much, I'd need to read the code to get this finer detail of what they mean

Reply:
Also, have you considered to use single logger, but use property accessors (i.e. dynamic properties) as a way to set proper events? feels like it will be simpler, and code churn would be smaller...

Reply:
The distinction is clear to me: Summarizer is the thing which summarizes, Summary is the summary being logged.

Dynamic property getters would be nice, but unfortunately they aren't as accurate when used in case of `refreshLatestSummaryAck`. There we need a way to verify that it is actually the same correlation (using summary sequence number or handle).
Maybe there's a way to get best of both worlds? Might be worth experimenting a little bit.

Reply:
Arin and I figured it out, thanks for the suggestion @vladsud